Sources/_CryptoExtras/AES/Block Function.swift

```diff
@@ -0,0 +1,125 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCrypto open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftCrypto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.md for the list of SwiftCrypto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Crypto
+@_implementationOnly import CCryptoBoringSSL
+@_implementationOnly import CCryptoBoringSSLShims
+@_implementationOnly import CryptoBoringWrapper
+import Foundation
+
+extension AES {
+    private static let blockSize = 128 / 8
+
+    public static func _permute<Payload: MutableCollection>(_ payload: inout Payload, key: SymmetricKey) throws where Payload.Element == UInt8 {
+        return try Self.permuteBlock(&payload, key: key, permutation: .forward)
+    }
+
+    public static func _inversePermute<Payload: MutableCollection>(_ payload: inout Payload, key: SymmetricKey) throws where Payload.Element == UInt8 {
+        return try Self.permuteBlock(&payload, key: key, permutation: .backward)
```

Comment:
Even though these are "private". Should we add some docs?

Reply:
Sure, one sec.

Reply:
Added, please re-review.